script.video.nfl.gamepass/addon.xml

```diff
@@ -20,12 +20,12 @@
     <license>GPL-2.0-only</license>
     <source>https://github.com/pigskin/kodi-gamepass</source>
     <disclaimer lang="en_GB">This addon requires you to have a subscription to NFL Game Pass.[CR]Please note that these subscriptions are region restricted by the NFL.[CR]This addon is completely unofficial and is /not/ endorsed by the NFL in any way.</disclaimer>
-    <news>2022.05.14
-    + Adapt to API changes implemented by Diva (jm-duke)
-    + Fix issue with LV Raiders logo (jm-duke)
-    + Fix display of 'None' for focused games in game week view (jm-duke)
-    + Implemented some improvements highlighted by Bas Rieter (jm-duke)
-    + Minor refactorings (jm-duke)
+    <news>2022.11.14
```

Comment:
Is this list correct? Seems like not all the changes are in the PR? If this was intentional, then NP.

Reply:
@jm-duke did you see this last comment?

Reply:
No, kinda missed it ;-)

I haven't updated the news list on purpose, because this PR just added a minor feature and introduced a quick fix for an API change.